flax2bnn.py

```diff
@@ -0,0 +1,157 @@
+"""Convert frequentist Flax modules to Bayesian NNs."""
+import operator
+from functools import reduce
+from typing import Optional
+
+import chex
+import flax.linen as nn
+import jax.numpy as jnp
+import numpyro.distributions as dist # ! Rather use jax.scipy distributions to avoid extra dependencies
```

Comment:
rather than using numpyro you should switch to the distributions that are shipped with jax.scipy. The extra dependency of numpyro is not necessary for you and should be avoided